src/boot/efi/meson.build

```diff
@@ -196,6 +196,10 @@ if have_gnu_efi
                                    '-j', '.dynsym',
                                    '-j', '.rel',
                                    '-j', '.rela',
+                                   '-j', '.rel.*',
+                                   '-j', '.rela.*',
+                                   '-j', '.rel*',
+                                   '-j', '.rela*',
```

Comment:
Why is both `.rel.*` and `.rel*` needed? Wouldn't the latter be enough?

Reply:
It didn't, with GNU objcopy (Gentoo 2.31.1 p5) 2.31.1.